api/ws/rpc/wsRPC.js

```diff
@@ -0,0 +1,188 @@
+'use strict';
+
+var MasterWAMPServer = require('wamp-socket-cluster/MasterWAMPServer');
+var scClient = require('socketcluster-client');
+var WAMPClient = require('wamp-socket-cluster/WAMPClient');
+var Q = require('q');
```

Comment:
Bluebird not installed and I was using Q every time before. Not a big thing as es6 migration is still on the roadmap. Should I change it?